snuba/settings/validation.py

```diff
@@ -30,6 +30,7 @@ def _validate_settings(locals: Mapping[str, Any]) -> None:
         "snuba-queries",
         "events-subscription-results",
         "transactions-subscription-results",
+        "sessions-subscription-results",
```

Comment:
This means Snuba will not be able to start if the topic is not present, do we really want that yet?